Update Wallet.md (#664)
btcpayserver/btcpayserver-doc: docs/Wallet.md

![Wallet Receive](./img/wallet/WalletReceive.png)

### Pull Payments

> This feature gives you the ability to create a Pull Payment, so that an outside individual may `pull` funds frop his wallet.

Reviewer: ```suggestion
This feature gives you the ability to create a Pull Payment, so that an outside individual may request to `pull` funds from your wallet.
```

> For more informations, see [Pull Payments](./pullpayments.md).

Reviewer: ```suggestion
For more information, see [Pull Payments](./PullPayments.md).
```

### Payouts

This section lets you manage Pull Payments and gives you the ability to accept or decline payouts requested by outside individuals.

> For more informations, see [Payouts](./pullpayments.md#approve-and-pay-a-payout).

Reviewer: ```suggestion
For more information, see [Payouts](./PullPayments.md#approve-and-pay-a-payout).
```
